pkg/virt-controller/watch/application.go

```diff
@@ -575,6 +575,9 @@ func (vca *VirtControllerApp) onStartedLeading() func(ctx context.Context) {
 
 		cache.WaitForCacheSync(stop, vca.persistentVolumeClaimInformer.HasSynced, vca.namespaceInformer.HasSynced, vca.resourceQuotaInformer.HasSynced)
 		close(vca.readyChan)
+		if err := metrics.RegisterLeaderMetrics(); err != nil {
+			golog.Fatal(err)
+		}
```

Comment:
maybe move this before the other metrics configuration: https://github.com/kubevirt/kubevirt/pull/12165/files#diff-603814ab7b1e6c7a99ebf35091dcb751056ad1d861bc45d424c89d8b799c6e1dR531-R543

Reply:
I'm not sure TBH.  I placed it based on the existing `metrics.SetVirtControllerLeading()`. In this location I can safely say it will run only by the leader pod.

Reply:
done